plugin/mutate/source/dextool/plugin/mutate/backend/database/type.d

```diff
@@ -225,3 +225,17 @@ struct MutantMetaData {
         attr.match!((NoMetadata a) {}, (NoMut a) => put(w, "nomut"));
     }
 }
+
+struct Rationale {
```

Comment:
Good that a specific Rationale type is created. It gives more information to to the programmer and the type system compared to a plain `string`.